src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -5,11 +5,9 @@ import cromwell.engine.ExecutionStatus.ExecutionStatus
 import cromwell.engine.backend.{Backend, JobKey}
 import cromwell.engine.db.slick._
 import cromwell.engine.workflow.{CallKey, ExecutionStoreKey, OutputKey}
-import cromwell.engine.{SymbolStoreEntry, WorkflowDescriptor, WorkflowId, WorkflowState}
+import cromwell.engine.{SymbolStoreEntry, WorkflowDescriptor, WorkflowId, WorkflowState, _}
```

Comment:
`import cromwell.engine._`, and move it up above the other engine imports (or the next committer optimizing imports will do it for you).